deployment/stacks/parent.ts

```diff
@@ -24,24 +24,12 @@ export class ParentStack extends Stack {
   ) {
     super(scope, id, props);
 
-    const {
-      stage,
-      project,
-      isDev,
-      vpcName,
-      bootstrapUsersPasswordArn,
-      oktaMetadataUrl,
-      brokerString,
-      iamPermissionsBoundaryArn,
-      iamPath,
-      deploymentConfigParameters,
-    } = props;
+    const iamPermissionsBoundaryArn = `arn:aws:iam::${process.env.CDK_DEFAULT_ACCOUNT}:policy/cms-cloud-admin/developer-boundary-policy`
```

Comment:
In other places we're using `${Aws.ACCOUNT_ID}` when I looked into this before, I landed on that as a preference.  If you disagree, we can switch, but I would like us to go all in on one or the other.